yt/utilities/parallel_tools/io_runner.py

```diff
@@ -173,7 +173,7 @@ def io_nodes(fn, n_io, n_work, func, *args, **kwargs):
 
     def gq(ds):
         dd = ds.all_data()
-        return dd.quantities["TotalQuantity"]("CellMassMsun")
+        return dd.quantities["TotalQuantity"](("gas", "cell_mass"))
```

Comment:
wow this one isn't trivial to me. Any insight ?

Reply:
As far as I could tell, `CellMassMsun` is mapped to `cell_mass` but I may be wrong. Either way, this is an example hidden in the `__main__` section of the file.

@matt I see you are one of the few authors of the file, what is its purpose? I don't see any reference in the code or the doc to it.